tests/sles4sap/publiccloud/general_patch_and_reboot.pm

```diff
@@ -37,7 +37,7 @@ sub run {
         kill_packagekit($instance);
         $instance->ssh_script_retry("sudo zypper -n --gpg-auto-import-keys ref", timeout => $ref_timeout, retry => 6, delay => 60);
         record_info('zypper ref time', 'The command zypper -n ref took ' . (time() - $cmd_time) . ' seconds.');
-        record_soft_failure('bsc#1195382 - Considerable decrease of zypper performance and increase of registration times') if ((time() - $cmd_time) > 240);
+        die('Considerable decrease of zypper performance and increase of registration times') if ((time() - $cmd_time) > 240);
```

Comment:
1. see line 36 `my $ref_timeout = check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE') ? 3600 : 240;` seems like you will have constant failures in Azure 
2.  I would expect a lot of failures here , are we sure that we want to go this way ? I mean yeah 4 minutes is a lot of time but there are many reasons why it could fail here but this failure can not be posted as a product bug ...

Reply:
Agree.  Wil drop this commit that would also affect QE-SAP.